src/org/opendatakit/briefcase/ui/ExportPanel.java

```diff
@@ -194,6 +205,20 @@ public void actionPerformed(ActionEvent e) {
     }
   }
 
+  class DateRangeActionListener implements ActionListener, DateChangeListener {
+    @Override
+    public void actionPerformed(ActionEvent e) {
+      pickStartDate.setEnabled(checkDates.isSelected());
+      pickEndDate.setEnabled(checkDates.isSelected());
+      ExportPanel.this.setEnabled(true);
```

Comment:
OK, I read above where you want to address this in another change. That’s fine with me.